fetch-utils.js

```diff
@@ -6,14 +6,20 @@ const client = supabase.createClient(SUPABASE_URL, SUPABASE_KEY);
 
 export async function getDogs() {
     // from the dogs table, select all items
-
+    const response = await client
+        .from('dogs')
+        .select();
```

Comment:
The functionality works perfectly, but you want to .select(*), to be explicit that Supabase should "select all."